wiki/Tournaments/MCNC/2019/en.md

```diff
@@ -0,0 +1,354 @@
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/908176)
+- [Livestream](https://live.bilibili.com/2996250)
+- [Challonge](https://challonge.com/zh_CN/MCNC)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1ii3yzv9lsReXbRN2dQTulp2izIpzSsp1zm_LIBVhzlQ)**
+
+## Participants
+
+| Number | osuID |
+| :-: | :-: |
+| 1 | ![][flag_CN] [Tamaki Iroha](https://osu.ppy.sh/users/11238501) |
+| 2 | ![][flag_CN] [- xiaoluoli -](https://osu.ppy.sh/users/950281) |
+| 3 | ![][flag_HK] [[Crz]Murasame](https://osu.ppy.sh/users/11563203) |
+| 4 | ![][flag_TW] [AradORiOn_MMM](https://osu.ppy.sh/users/7960927) |
+| 5 | ![][flag_CN] [Kakuko](https://osu.ppy.sh/users/10413362) |
+| 6 | ![][flag_TW] [AW ZERO](https://osu.ppy.sh/users/4415146) |
+| 7 | ![][flag_TW] [Sakaki](https://osu.ppy.sh/users/2656856) |
+| 8 | ![][flag_CN] [[RT]Alleyne](https://osu.ppy.sh/users/11279273) |
+| 9 | ![][flag_CN] [[Crz]Satori](https://osu.ppy.sh/users/7082178) |
+| 10 | ![][flag_CN] [[Crz]HDRoop_7](https://osu.ppy.sh/users/6336721) |
+| 11 | ![][flag_CN] [Lo_liFish2000](https://osu.ppy.sh/users/8828244) |
+| 12 | ![][flag_CN] [snowkyou](https://osu.ppy.sh/users/12513448) |
+| 13 | ![][flag_CN] [Namida KL](https://osu.ppy.sh/users/6889190) |
+| 14 | ![][flag_CN] [[Crz]I-Night](https://osu.ppy.sh/users/8790619) |
+| 15 | ![][flag_CN] [LFWY](https://osu.ppy.sh/users/8611484) |
+| 16 | ![][flag_TW] [[Crz]FolAH1217](https://osu.ppy.sh/users/6232458) |
+| 17 | ![][flag_CN] [ying huo](https://osu.ppy.sh/users/9077996) |
+| 18 | ![][flag_CN] [ImmortalphoeniX](https://osu.ppy.sh/users/7080633) |
+| 19 | ![][flag_CN] [Pureforever](https://osu.ppy.sh/users/11943671) |
+| 20 | ![][flag_CN] [[RT]Mrqianmo](https://osu.ppy.sh/users/10333067) |
+| 21 | ![][flag_CN] [[Crz]Serika](https://osu.ppy.sh/users/6701729) |
+| 22 | ![][flag_CN] [tyrcs](https://osu.ppy.sh/users/13026904) |
+| 23 | ![][flag_CN] [Ecivon](https://osu.ppy.sh/users/9430102) |
+| 24 | ![][flag_CN] [Stink God](https://osu.ppy.sh/users/7381269) |
+
+## Groups
+
+| Group A | Group B | Group C | Group D | Group E | Group F | Group G | Group H |
+| :-- | :-- | :-- | :-- | :-- | :-- | :-- | :-- |
+| AradORiOn_MMM | [Crz]Murasame | [Crz]FolAH1217 | [Crz]Satori | tyrcs | [Crz]HDRoop_7 | Kakuko | Stink God |
+| LFWY | [RT]Alleyne | [Crz]I-Night | ImmortalphoeniX | [Crz]Serika | AW ZERO | Sakaki | Pureforever |
+| Lo_liFish2000 | Tamaki Iroha | [RT]Mrqianmo | Namida KL | snowkyou | Ecivon | - xiaoluoli - | ying huo |
+
+---------------------------
+
+- 1st place : [Crz]Murasame
+- 2nd place : [Crz]FolAH1217
+- 3rd place : [Crz]Satori
+
+![MCNC 2019 Bracket](img/Bracket.png)
+
+---------------------------
+
+## Mappools
+
+### Finals
+
+**[Download the mappack here! (154MB)](https://www.dropbox.com/s/r4p7tbxa58ijgzh/MCNC19%20%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [TAG underground overlay - Over The "Period" (Tidek) [Edit]](https://osu.ppy.sh/b/1777377)
```

Comment:
```suggestion
  - [TAG underground overlay - Over The "Period" (Tidek) \[Edit\]](https://osu.ppy.sh/beatmaps/1777377)
```
Use latest linking. Apply for all /b/
And escape all brackets used to wrap the difficulty name `\[Edit\]`. Apply for all too.